Description: Spelling Errors
 Fix some minor spelling errors in the code
Author: Net-SNMP Packaging Team <pkg-net-snmp-devel@lists.alioth.debian.org>
Reviewed-by: Craig Small <csmall@debian.org>
Last-Update: 2021-09-20

--- a/apps/snmpnetstat/inet.c
+++ b/apps/snmpnetstat/inet.c
@@ -456,7 +456,7 @@
     struct stat_table icmpstats_tbl[] = {
         {1, "%14lu total message%s received"},
         {2, "%14lu message%s dropped due to errors"},
-        {14, "%14lu ouput message request%s"},
+        {14, "%14lu output message request%s"},
         {15, "%14lu output message%s discarded"},
         {0, ""}
     };
--- a/apps/snmpnetstat/inet6.c
+++ b/apps/snmpnetstat/inet6.c
@@ -332,7 +332,7 @@
     struct stat_table icmp6stats_tbl[] = {
         {1, "%14d total message%s received"},
         {2, "%14d message%s dropped due to errors"},
-        {18, "%14d ouput message request%s"},
+        {18, "%14d output message request%s"},
         {19, "%14d output message%s discarded"},
         {0, ""}
     };
